analytics/analytics/docs/redoc.html

```diff
@@ -2,26 +2,21 @@
 <html>
   <head>
     <title>Openverse Analytics</title>
-    <!-- needed for adaptive design -->
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    <link rel="icon" type="image/svg" href="https://raw.githubusercontent.com/WordPress/openverse/master/brand/icon.svg">
     <!--
     ReDoc doesn't change outer page styles
     -->
     <style>
-      body {
+    body {{
         margin: 0;
         padding: 0;
-      }
-
-      img[alt='logo'] {
-        padding: 20px; /* same as other sidebar items */
-      }
+    }}
```

Comment:
Whoops! Looks like an accidental edit here adding unnecessary extra curly braces.

Reply:
Spectree (the Python plugin we're using to generate ReDoc) reads the file as a string and uses the `.format` function to populate the Swagger URL. To prevent Python from trying to interpolate the contents of `{ }`, we must [double the braces](https://stackoverflow.com/a/5466478) so that Python treats them as literals.

Reply:
😮 That's interesting. Should we really use `html` as the file extension then? That's pretty surprising behavior for an html file that doesn't really have any other indiciation that it's actually a template string... Maybe that's just me though.